lib/containers/container_images.pm

```diff
@@ -130,6 +123,40 @@ sub test_opensuse_based_image {
     assert_script_run("$runtime run --rm refreshed-image sh -c 'zypper -v ref | grep \"All repositories have been refreshed\"'", 120);
 }
 
+# Testing sle based images
+sub test_sle_based_image {
```

Comment:
This is just code duplication. We already have `test_opensuse_based_image ` which handles also SLE and works just fine.

Reply:
think open/close principle. indeed `test_opensuse_based_image` handles both but it is hard to maintain. there are differences with sle base and this make it simplier to follow. i think all the confusion came from here in the first place.

Reply:
I'm with pdostal here :)

Reply:
i do not disagree that it has duplicate code. but it makes more simple to tackle what will run against sle base and what with anything else. i could go further to reduce the duplication but it was not my main concern. The main problem was what it runs when. as now opensuse will be used in both o3 and osd and there was conditions that it was hard to follow.